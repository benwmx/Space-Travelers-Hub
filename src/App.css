@import url('https://cdn-ec853.web.app/css/style.css');

.btn {
  border-radius: 8px;
  background-color: rgb(30, 144, 255);
  color: #ffffff;
}