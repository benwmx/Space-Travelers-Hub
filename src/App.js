import React from 'react';
import { HashRouter as Router, Route, Switch } from 'react-router-dom';
import Header from './components/Header';
import Missions from './components/Missions';
import Rockets from './components/Rockets';
import MyProfile from './components/MyProfile';
import './App.css';

function App() {
  return (
    <Router>
      <Header />
      <Switch>
        <Route path="/" exact>
          <Rockets />
        </Route>
        <Route path="/missions">
          <Missions />
        </Route>
        <Route path="/myprofile">
          <MyProfile />
        </Route>
      </Switch>
    </Router>
  );
}

export default App;
