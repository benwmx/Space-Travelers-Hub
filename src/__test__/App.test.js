import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import App from '../App';
import store from '../redux/configureStore';

const APP = (
  <Provider store={store}>
    <App />
  </Provider>
);

describe('App Component test', () => {
  it('Should render', () => {
    const app = renderer.create(APP).toJSON();
    expect(app).toMatchSnapshot();
  });
});
