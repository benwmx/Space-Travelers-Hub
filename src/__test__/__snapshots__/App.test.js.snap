// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component test Should render 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="logo"
    >
      <img
        alt="Yellow Planet"
        src="logo.svg"
        style={
          Object {
            "height": "50px",
            "width": "50px",
          }
        }
      />
      <h1
        className="title"
      >
        Space Travelers' Hub
      </h1>
    </div>
    <div
      className="navLinks"
    >
      <a
        aria-current="page"
        className="activeLink"
        href="#/"
        onClick={[Function]}
        style={Object {}}
      >
        Rockets
      </a>
      <a
        aria-current={null}
        href="#/missions"
        onClick={[Function]}
      >
        Missions
      </a>
      <a
        aria-current={null}
        href="#/myprofile"
        onClick={[Function]}
      >
        My profile
        <span
          className="numberOfSelected"
        >
          R(
          0
          ) M (
          0
          )
        </span>
      </a>
    </div>
  </header>,
  <div
    className="rocketsContainer"
  >
    <div
      className="rocket"
    >
      <div>
        <img
          alt="Rocket"
          className="image"
          src=""
        />
      </div>
      <div
        className="info"
      >
        <h2>
          
        </h2>
        <p
          className="description"
        >
          <span />
          
        </p>
        <button
          className="reserveRocket"
          id=""
          onClick={[Function]}
          type="button"
        >
          Reserve Rocket
        </button>
      </div>
    </div>
  </div>,
]
`;
